src/util/postgres_client.js

```diff
@@ -742,10 +742,27 @@ class PostgresTable {
 
     async find(query, options = {}) {
 
+        function isObject(v) {
+            return (typeof v === 'object' && !Array.isArray(v) && v !== null);
+        }
+        function calculateOptionsAndArrayFields(q) {
+            const ops = [ '$all' ];
```

Comment:
the handling of `$all` operator should be in `handle_ops_encoding`. see for example how `$in` is handled.

Reply:
👍